include/oneapi/dpl/internal/dynamic_selection_impl/concurrent_queue.h

```diff
@@ -0,0 +1,55 @@
+// -*- C++ -*-
+//===---------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
```

Comment:
`pragma once` without guard. Looks inconsistent. I guess it's "special" header since it doesn't have namespaces, but still looks odd to me

Reply:
We should align with oneDPL. Use of "pragma once" came from PoC.